src/node.cc

```diff
@@ -1735,18 +1757,21 @@ static gid_t gid_by_name(Isolate* isolate, Handle<Value> value) {
 
 static void GetUid(const FunctionCallbackInfo<Value>& args) {
   // uid_t is an uint32_t on all supported platforms.
+  ScopedLock::Read lock(&process_rwlock);
```

Comment:
Doesn't need a lock, thread safety is guaranteed by libuv and the operating system.